app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -151,6 +150,13 @@ export class LabEntryForm extends PureComponent {
     }
   }
 
+  discardTestInDraft = (test, action) => {
+    const { dispatch } = this.props;
+    if (action === 'single') return dispatch(removeTestFromDraft(test));
+    if (action === 'panel') return dispatch(removeTestPanelFromDraft(test));
+    return dispatch(deleteDraftLabOrder());
+  }
```

Comment:
The variable name `discardTestInDraft` implies that this function is meant to remove a test from the draft but from the implementation details, I see that there is a possibility of this function deleting the draft itself.
I suggest you separate the two functionalities in different functions or better still you make the name of this function reflect what it really does.

Reply:
@IAMOTZ  done